canonical-web-and-design/maas-ui
# feat(ui): build add logical volume form

## Done

- Built the "Add logical volume" form

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the angular version of a machine's storage tab
- Create one or more unmounted partitions
- Select the partitions and then click "Create volume group" and confirm
- Go to the react version of the machine's storage tab and open the action menu for the volume group
- Click "Add logical volume..."
- Check that you can successfully add a logical volume by submitting the form

## Fixes

Fixes canonical-web-and-design/MAAS-squad#2165
